src/webgpu/web_platform/canvas/readbackFromWebGPUCanvas.spec.ts

```diff
@@ -329,3 +329,13 @@ g.test('drawTo2DCanvas')
 
     readPixelsFrom2DCanvasAndCompare(t, ctx, expect[t.params.alphaMode]);
   });
+
+g.test('transferToImageBitmap_unconfigured')
+  .desc(`Regression test for a crash when calling transferImageBitmap on an unconfigured canvas`)
+  .fn(t => {
+    const canvas = createCanvas(t, 'offscreen', 2, 2);
+    canvas.getContext('webgpu');
+    t.shouldThrow(true, () => {
+      canvas.transferToImageBitmap();
```

Comment:
I think this is supposed to return a blank image instead of throwing? See "When transferToImageBitmap() is called on a canvas with GPUCanvasContext context"

Reply:
What should happen when the canvas size is empty? It looks it is no possible to create an empty image bitmap. Should we return null in that case?

Reply:
Tentatively assuming it should raise an exception.

Do you know the best way to check the contents of an ImageBitmap?

Reply:
An exception seems correct for Wx0 or 0xH.

To check an imagebitmap I think you'll have to draw it into something, for example 2d canvas drawImage.